### PR TITLE
feature 604: implement datePicker to schedule posts

### PR DESCRIPTION
**Pull request by anechytailo**
### Type of Pull Request *
- [ ] CHANGE (fix or feature that would cause existing functionality to not work as expected)
- [x] FEATURE (non-breaking change which adds functionality)
- [ ] BUGFIX (non-breaking change which fixes an issue)
- [ ] ENHANCEMENT  (non-breaking change which improves existing functionality)
- [ ] NONE (if none of the other choices apply. For example, tooling, build system, CI, docs, etc.)

### Related links
 
**Issue link**

 
**Story link**
[#604 Story](https://github.com/ita-social-projects/dokazovi-requirements/issues/604)
 
### Description *
Create Date and time picker to schedule publish time.
 
### Summary of change
 
"*" - mandatory field

### CODE REVIEW

File: src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx [hunk 1]
```diff
@@ -43,13 +43,13 @@ const ChangePublicationDateModal: React.FC<IChangeViewsCountModal> = ({
...
-        {posts[id].createdAt}
+        {posts[id].publishedAt}
       </Typography>
       <Typography>{t(langTokens.admin.newPublicationDate)}</Typography>
       <DatePicker
-        value={newPostPublicationDate.split('.').reverse().join('-')}
+        value={new Date(newPostPublicationDate)}
         variant="inline"
-        format="dd.MM.yyyy"
+        format="dd.MM.yyyy  HH:mm"
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx:52]: why 2 spaces?

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    right: '14px',
+    bottom: '16px',
+  },
+
+  datePicker: {
+    position: 'relative',
+
+    '& > label': {
+      backgroundColor: theme.palette.common.white,
+      margin: '8px 0 0 18.5px',
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx:20]: may we remove fractional part here?

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  selectAdminLab,
+  selectModifications,
+  setNewPostDateInput,
+} from '../../../../models/adminLab';
+import { useActions } from '../../../../shared/hooks';
+
+interface IChangeViewsCountModal {
+  id: number;
+}
+const dateRangeIcon = (
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx:18]: propose to extract SVG icon into separate file

File: src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx (hunk 1 above)
Comment (anechytailo) [reply src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx:52]: Typo((

**Comment by anechytailo** [comment]
@skyboyer please review

**Review by skyboyer — APPROVED** [review]
nice and clean. like it.

**Comment by anechytailo** [comment]
@VitaliiKulinskyi please approve

**Review by VitaliiKulinskyi — APPROVED** [review]

**Pull request merged**